index.html

```diff
@@ -52,6 +54,164 @@ <h1> Hey There. <br> I'm Dennis <br>
           </ul>
         </div>
       </section> 
+      
+      <section id="recentWorks">
+        <div class="imgHeader">
+          <h2>My Recent Works</h2>
+          <img src="./images/imgPlaceholder.png" alt="placeholderImg">
+        </div>
+        <div id="works">
+          <div class="introInfo">
+            <div class="title"><span>Multi-Post Stories</span></div>
+            <p>
+              <span>A daily selection of privately personalized reads; no accounts or sign-ups required. 
+              has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+              </span>
+            </p>
+          </div>
+          <div class="buttonGroup">
+            <div class="button">css</div>
+            <div class="button">html</div>
+            <div class="button">bootstrap</div>
+            <div class="button">Ruby</div>
+          </div>
```

Comment:
- [ ] For The list of project technologies (e.g., "HTML" and "CSS") Kindly I recommend you to use `ul` tag to group them, and `li` per icon for list of project technologies; it will make your HTML code semantically correct